### ISSUE TITLE
Validate healthsite data

### ISSUE THREAD

**Issue by markherringer** [healthsites/healthsites#962]
As a trusted user i want to publish updates to Healthsites so that I can help to validate the data.

**Comment by litlfred** [comment]
Will there be a "holding area" for the updates that are published until they are verified / validated / de-spammified?

Cheers,
-carl

**Comment by markherringer** [comment]
Yes, here are user stories that are close to the one you are describing:
https://github.com/healthsites/healthsites/issues/848
https://github.com/healthsites/healthsites/issues/841

**Comment by litlfred** [comment]
Thanks.  Flagging for @lduncan-intrah (or at least trying to!)
